ui/litellm-dashboard/src/components/mcp_tools/mcp_connect.tsx

```diff
@@ -101,6 +102,24 @@ const FeatureCard: React.FC<FeatureCardProps> = ({
             />
             <Text className="text-sm">Segregate tools to just use {serverName} tools</Text>
           </div>
+          {useServerHeader && (
```

Comment:
same on the UI - only show how to use csv, not the old array

Reply:
ack